app/scripts/modules/core/cluster/filter/clusterFilter.component.ts

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+import {CLUSTER_FILTER_SERVICE} from 'core/cluster/filter/clusterFilter.service';
+
+import {compact, uniq, map} from 'lodash';
+import {IScope, module} from 'angular';
+import {Application} from '../../application/application.model';
+
+export const CLUSTER_FILTER = 'cluster';
```

Comment:
This was the module name in the existing code.  I also found it odd that `cluster` was used as a module name in the "cluster filter component" (instead of in some file in the parent directory).  Will update the name in the filter component.